doc/markdown/plannerTerminationConditions.md

```diff
@@ -0,0 +1,23 @@
+# Planner Termination Conditions {#plannerTerminationConditions}
+
+OMPL uses the ompl::base::PlannerTerminationCondition concept to let planning algorithms know that an attempt to solve a motion planning problem should be terminated. This is done by passing an instance of an ompl::base::PlannerTerminationCondition to a planner's solve method. A PlannerTerminationCondition instance is essentially a functor that can be called with zero arguments that returns `true` when the planner should terminate. For convenience, there is also a cast-to-bool operator defined so the following two if statements are equivalent:
+
+~~~{.cpp}
+// ptc is an instance of ompl::base::PlannerTerminationCondition
+if (ptc()) return;
+if (ptc) return;
+~~~
+
+OMPL implements several different conditions that you can use:
+
+- `timedPlannerTerminationCondition(duration)`: a **function** that returns a PlannerTerminationCondition instance that causes the planner to terminate after a fixed amount of time has been exceeded.
+- `timedPlannerTerminationCondition(duration, interval)`: a **function** that returns a PlannerTerminationCondition instance that causes the planner to terminate after a fixed amount of time has been exceeded. Unlike the previous function, this one spawns a separate thread where the time passed is checked every `interval` seconds. 
+- `IterationTerminationCondition(numIterations)`: a PlannerTerminationCondition-derived **class** that causes a planner to terminate after its `eval()` method has been called `numIterations` times.
+- `CostConvergenceTerminationCondition(pdef, solutionsWindow, convergenceThreshold)`: a PlannerTerminationCondition-derived **class** that causes an *optimizing* planner  to terminate once the cost for the best path has converged, which is defined as 
+- `exactSolnPlannerTerminationCondition()`: a **function** that returns a PlannerTerminationCondition instance that causes the planner to terminate after an exact solution has been found.
```

Comment:
```suggestion
- `exactSolnPlannerTerminationCondition()`: a **function** that returns a PlannerTerminationCondition instance that causes the planner to terminate after an exact (not approximate) solution has been found.
```

Without that, I always start to wonder if exact means optimal, or resolution exact, or something else.